docs/modis_SimCLR/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM nvidia/cuda:11.2.1-runtime-ubuntu20.04
```

Comment:
There is a folder named `dockers`.
Please put this in there and name it something
specific.  Maybe `simclr_v1.docker` ?